pagopa/pn-frontend
# refactor: validate IUN after submitting the notification filter form

## Short description
Validate IUN after submitting the notification filter form

## List of changes proposed in this pull request
- Changed validation mode

## How to test
In PA, try to insert an invalid IUN and click the 'apply' button. You should see the error
